lib/components/Stepper/types.ts

```diff
@@ -0,0 +1,13 @@
+import { TypographyProps } from "@components/Typography/base";
+
+type TEnum = { [key: string]: number };
+export type TTypographySizes = Extract<TypographyProps['size'], 'sm' | 'md' | 'lg'>;
+
+export type TBasicStepperProps = {
+    currentStep: number | TEnum;
+    Icon?: ({ size }: { size: TTypographySizes }) => React.ReactNode;
+    size?: TTypographySizes;
+    labels: string[];
+    lineSize?: string;
+    orientation?: 'vertical' | 'horizontal';
```

Comment:
Can we move this type to `lib/types.ts` so we can reuse it later?

Reply:
done